apps/etterlatte-saksbehandling-ui/client/src/components/person/saksoversikt.tsx

```diff
@@ -145,10 +145,14 @@ export const Saksoversikt = ({ fnr }: { fnr: string | undefined }) => {
                     disabled={harAapenRevurdering}
                     grunnlag={personerISak}
                   />
-                  <div className="behandlinger">
-                    <h2>Behandlinger</h2>
-                    {behandlingliste !== undefined && <Saksliste behandlinger={behandlingliste} />}
-                  </div>
+                  {sakId !== undefined ? (
```

Comment:
Unødvendig ternary, så kan forenkles litt. Undefined-sjekk på linje 151 kan også forenkles til `!behandlingliste` 

```
{!sakId && (<>)}
```

Reply:
Tenkte at det var greit å gjøre likt som den andre sjekken, @oyvindsh du er vel ikke helt enig i `&&` ? Begge kan jo fint brukes.

Reply:
Blir litt hipp som happ, men jeg foretrekker eksplisitt kontroll med ternary, ref [https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)